CHANGELOG.md

```diff
@@ -138,3 +138,19 @@
  * Change JweAlgorithm to sealed class to support unknown algorithms
  * Add generic `ECPoint` class
  * Implement elliptic-curve arithmetic
+
+
+### 3.2.0
+
+* Kotlin 2.0
+* Gradle 8.8
+* Bouncy Castle 1.78.1
+* Kotest 5.9.1 
+* Coroutines 1.8.1
+* Serialization 1.7.1-SNAPSHOT
+
+**Fixes**
+ * Move `curve` from `CryptoAlgorithm` to `JwsAlgorithm`
+ * Don't read curve information into the X.509 tea leaves where none exists
+   * `CryptoSignature`s in X.509 are now indefinite length
+ * Rename `CryptoAlgorithm` to `X509SignatureAlgorithm` to better describe what it is
```

Comment:
because what's contained in there conforms to X509 (in contrast to JWS oder COSE).
The key "missing" information for EC signatures, for example, is the curve and our encoding/decoding function for what used to be called. `CryptoSignature` were strictly following X509, so @iaik-jheher convinced me that a rename was in order. Also: we are only dealing with signatures in this class. So if we expand this to encryption, it makes perfect sense to separate this (like we do with JWA already)